dummy/src/toga_dummy/window.py

```diff
@@ -92,7 +94,8 @@ def close(self):
         self._set_value("visible", False)
 
     def get_image_data(self):
-        return b"pretend this is PNG image data"
+        self._action("get image data")
+        return (toga.App.app.paths.app / "resources/toga.png").read_bytes()
```

Comment:
To avoid confusion during testing, it might be worth generating a dummy small "window" image so nobody gets confused about what is being returned here. If that image is a resource in the dummy app, it can also be referenced without needing the app as a fixture.

Reply:
Okay, I put a dummy screenshot in a new dummy/src/toga_dummy/resources folder. This is probably a dumb question, but what path do I use to access that at runtime? Also, is there an inventory somewhere I need to list it in to fix the "lists of files in version control and sdist do not match!" error?

Reply:
Right now, the file needs to be matched by the patterns contained in MANIFEST.in for the package where the file is contained. If you run `check-manifest` over the package (which will be done by `toga -e package -- dummy`), it will tell you a candidate line to add.

However - once #2229 lands, that won't be required.

As for the path to reference the file - `toga_dummy.__file__` should be a safe anchor to base a path from. That's effectively what `app.paths.toga` does (except with `toga.__file__`).

Reply:
Thank you — I'm able to construct the right paths now, but I'm confused by MANIFEST.in's behavior. I can run `check-manifest` and it recommends I add `recursive-include src *.png`. Fair enough. I do, and `check-manifest `(as well as `tox -e packge -- dummy`) are happy. But when I run the core tests, even after completely deleting tox's `py` environment in case of caching problems, the generated wheels still don't include the added images in dummy.

Looking at the core MANIFEST.in for comparison just made me more confused; it only lists image includes for `tests`, but its wheel gets the images in `src` just fine. I must be missing something here.

Reply:
.....How close do you think that other PR is? 😅

Reply:
My hope would be early next week - so, hopefully only a day or two?

Reply:
FYI - #2229 has now landed.